systemd/ua-license-check.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Ubuntu Advantage Timer to check for GCP Ubuntu Pro licenses
+
+[Timer]
+OnCalendar=*:0/5
+RandomizedDelaySec=5min
+Persistent=true
```

Comment:
We might want to drop the persistent=true. This has a side-effect that if the machine was off for > 5 minutes. It'll be run ASAP on next boot... which will likely occur before OnStartupSec=1min and increase our chances of colliding with a either lock held by ua-auto-attach, or with cloud-init trying to detect the datasouce (for cloud-id).  Also any objection to increasing OnStartupSec to > 2 mins for sort of the same reason. Cut down on risk area where collisions of multiple UA config writers in initial system boot.

Reply:
Done!